### PR TITLE
Amsua bugfixes

### PR DESCRIPTION
**Pull request by jswhit**
Addressed bug reports related to the amsua channels documented by @frolovsa in [this document](https://docs.google.com/document/d/1DjkKN27jDB7FqG2aT5T04XJ49kIT4wjsYTbbL7lD13k/edit?pli=1#). The follow up PR will address inconsistencies in white/blacklist selections between noaa and nasa"

### CODE REVIEW

File: satinfo/amsua_metop-b/readme [hunk 1]
```diff
@@ -1,2 +1,3 @@
-metop-b 20130213 000000  20161108 060000  amsua 15  1 2 3 4 5 6 7 8 9 10 11 12 13 14 15     # start as in Merra
-metop-b 20161108 120000  21001231 240000  amsua  8              7 8 9 10 11 12 13 14       # turn off 1-6 & 15 as in operational satinfo (Merra turns off 20160925)
+metop-b 20130213 000000  20151114 060000  amsua 15  1 2 3 4 5 6 7 8 9 10 11 12 13 14 15     # start as in Merra
+metop-b 20151114 120000  20161108 060000  amsua 15  1 2 3 4 5 6   8 9 10 11 12 13 14 15     # NCEP turned off ch7
```
Comment (frolovsa) [inline satinfo/amsua_metop-b/readme]: looks like "amsua 15" is wrong. should it state "amsua 14" instead?

File: satinfo/amsua_n15/readme [hunk 2]
```diff
@@ -4,5 +4,5 @@ n15  19990101 060000  19990101 060000  amsua  0							# off due to noisy (as in
 n15  19990101 120000  20001029 240000  amsua 15  1 2 3 4 5 6 7 8 9 10 11 12 13 14 15   		# channel 11 off (as in CFSR; Merra turns it off 20020401)
```
Comment (frolovsa) [inline satinfo/amsua_n15/readme]: still need to remove "11" from this line and set number of assimilated channels to 14.

File: satinfo/sndrD_g15/2020102200 [hunk 3]
```diff
@@ -0,0 +1,72 @@
+ sndrD1_g15              1  -1    0.900    0.000    2.000   10.000    0.000      1     -1     -1
```
Comment (frolovsa) [inline satinfo/sndrD_g15/2020102200:1]: Jeff did this sndrd file creep in to tis PR? I am not sure what is the rationale for it. Maybe better remove it from this PR and add it in a separate PR that documents why this has been done.

Comment (frolovsa) [reply satinfo/sndrD_g15/2020102200:1]: I also see that you added readme file saying there are no dumps after this date. Did you mean for this file to be all "off". As is this file has a bunch of channels turned on.

**Review by frolovsa — CHANGES_REQUESTED** [review]
Thank you Jeff, for working through my comments. I found a couple of inconsistencies (see inline notes). I will also edit the description of this PR to post a link to the document.

**Comment by frolovsa** [comment]
@jswhit looks like you are the only one who can edit the PR description ca you add something like this:
"Addressed bug reports related to the amsua channels documented by @frolovsa in [this document](https://docs.google.com/document/d/1DjkKN27jDB7FqG2aT5T04XJ49kIT4wjsYTbbL7lD13k/edit?pli=1#). The follow up PR will address inconsistencies in white/blacklist selections between noaa and nasa"

File: satinfo/amsua_metop-b/readme (hunk 1 above)
Comment (jswhit) [reply satinfo/amsua_metop-b/readme]: done

File: satinfo/amsua_n15/readme (hunk 2 above)
Comment (jswhit) [reply satinfo/amsua_n15/readme]: moved comment on Ch 11 to next line (Ch11 turned off on 20000130)

File: satinfo/sndrD_g15/2020102200 (hunk 3 above)
Comment (jswhit) [reply satinfo/sndrD_g15/2020102200:1]: yes, accidemtally branched from a branch with these changes.

**Review by frolovsa — APPROVED** [review]
Thank you Jeff. looks good. Ready to merge from my perspective.

**Pull request merged**